lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -334,7 +334,7 @@ defmodule Phoenix.LiveViewTest do
       end
 
     start_proxy(path, %{
-      html: Phoenix.ConnTest.html_response(conn, 200),
+      html: format_response(conn, 200),
```

Comment:
I go down this route, then we should probably have this:

```suggestion
      response: Phoenix.ConnTest.response(conn, 200),
```

And pass the format as a separate option to the proxy.

Reply:
changing `html` to `response` kicks off a ton of changes, going down into the proxy itself.